main/realmain.cc

```diff
@@ -164,15 +166,6 @@ core::Loc findTyped(unique_ptr<core::GlobalState> &gs, core::FileRef file) {
 }
 
 #ifndef SORBET_REALMAIN_MIN
-class LSPStdout final : public lsp::LSPOutput {
```

Comment:
I'm a little embarrassed... in a previous PR I moved this class into `LSPOutput.{h,cc}`, but must've forgotten to remove it from realmain.